src/repl/scala/tools/nsc/interpreter/session/FileBackedHistory.scala

```diff
@@ -80,5 +80,5 @@ object FileBackedHistory {
   import Properties.userHome
 
   def defaultFileName = ".scala_history"
-  def defaultFile: File = File(Path(userHome) / defaultFileName)
+  def defaultFile: File = File( Path( Properties.propOrElse( "scala.repl.histfile", userHome +"/"+ defaultFileName) ) )
```

Comment:
Ok, all modern Windows handle it fine, and the java File API normalizes everything that comes through the public API anyway, so it all works (I can't believe I am complimenting the Java 6 File API). Just ignore my comments :-)